app/services/notices/setup/fetch-returns-due-by-licence-ref.service.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+/**
+ * Fetches the returns due for `/notices/setup/{sessionId}/returns-for-paper-forms` page
+ * @module FetchReturnsDueByLicenceRefService
+ */
+
+const ReturnLogModel = require('../../../models/return-log.model.js')
```

Comment:
Should we replace the existing validation with this. 

And save this into the session ?